hbase-client/src/main/java/org/apache/omid/transaction/HTableAccessWrapper.java

```diff
@@ -20,10 +20,7 @@
 import java.io.IOException;
 import java.util.List;
 
-import org.apache.hadoop.hbase.client.Get;
-import org.apache.hadoop.hbase.client.HTableInterface;
-import org.apache.hadoop.hbase.client.Put;
-import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.client.*;
```

Comment:
Was the use of wildcarding here intentional? Not sure about Omid, but in Phoenix we always explicitly specify all the imports.